com.fightcade.Fightcade.desktop

```diff
@@ -0,0 +1,12 @@
+[Desktop Entry]
```

Comment:
There is a desktop file provided by the upstream if it's preferable that I use that.

```
#!/usr/bin/env xdg-open
[Desktop Entry]
Version=1.0
Type=Application
Terminal=false
Exec=sh -e -c "exec \\"\\$(dirname \\"\\$0\\")/Fightcade2.sh\\"" %k
Name=Fightcade
Comment=Fightcade
Categories=Game;Emulator;ArcadeGame
Icon=
```